src/test/java/org/broadinstitute/consent/http/service/UseRestrictionConverterTest.java

```diff
@@ -234,4 +238,85 @@ public void testTranslateTrueValues() {
     assertTrue(dataUse.getHmbResearch());
   }
 
+  @Test
+  public void testTranslateDataUseSummary() {
```

Comment:
I had to add it because we weren't ever calling the data use summary endpoint in consent, so I had to add the plumbing in `UseRestrictionCoverter` - which felt like an appropriate place, but I could be wrong. The correspond code was added in UseRestrictionConverter

Reply:
Is there a place we put ontology calls? It feels weird to lump it into the rest of the service class I added, but also I don't see anywhere else appropriate.